compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/a11y/ComposeAccessible.kt

```diff
@@ -332,9 +332,13 @@ internal class ComposeAccessible(
         }
 
         override fun getAccessibleChild(i: Int): Accessible? {
-            return semanticsNode.replacedChildren.getOrNull(i)?.id?.let { id ->
-                controller.accessibleByNodeId(id)
-            } ?: auxiliaryChildren[i - semanticsNode.replacedChildren.size]
+            val replacedChildren = semanticsNode.replacedChildren
+            val replacedChildrenSize = replacedChildren.size
+            return if (i < replacedChildrenSize) {
+                controller.accessibleByNodeId(replacedChildren[i].id)
```

Comment:
The only difference I see is using `get` instead of `getOrNull`. I don't really get why it should make a difference.

Reply:
It's a silly mistake that's not likely to be repeated, and it's hard to set up a test for this. I don't think it's worth it.